.github/workflows/build-all-arduino.yml

```
@@ -0,0 +1,85 @@
+on:
+  push:
+    tags:
+      - '*.*.*'  # Matches date format tags like dd.mm.yyyy
+
+jobs:
+  build-and-release:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    steps:
+    - name: 📦 Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)